src/types.h

```diff
@@ -67,6 +68,7 @@ class SizedType
     stack_type = stack_type_;
   }
   Type type;
+  Type elem_type;
```

Comment:
Could you add a comment describing what this field is for please?

I was pretty lax about commenting stuff like this, but the codebase is getting bigger and more people are starting to work on it now so it's getting harder to keep track of everything.

Reply:
Done